app/views/finance/financial_transactions/_transactions.html.haml

```diff
@@ -20,6 +20,11 @@
       - if FinancialTransactionType.has_multiple_types
         %th= heading_helper FinancialTransaction, :financial_transaction_type
       %th= sort_link_helper heading_helper(FinancialTransaction, :note), "note"
+      %th= heading_helper FinancialTransaction, :payment_plugin
+      %th= heading_helper FinancialTransaction, :payment_method
+      %th= heading_helper FinancialTransaction, :payment_amount
+      %th= heading_helper FinancialTransaction, :payment_fee
+      %th= heading_helper FinancialTransaction, :payment_state
```

Comment:
Is the table still readable with a number of added columns (with possibly long names)?
This is especially relevant for foodcoops not using a payment plugin.

Reply:
If we don't need a config option, let's avoid it (as it's complex enough as it is). So if they're a straightforward way to derive this, yes. Looking at payment_plugin config seems sensible here, yes.

Could there be a situation where this information is still relevant but the payment plugin is disabled? E.g. a payment plugin was enabled, but now disabled? I think we can just hide it, because any payment-plugin-related things will fail anyway. But just checking. E.g. at some places we check if there's more than one financial transaction type, and only show certain things if there is (to avoid UI complexity when a feature is not used).